analyzer/os/ubuntu/ubuntu_test.go

```diff
@@ -14,36 +15,91 @@ import (
 
 func Test_ubuntuOSAnalyzer_Analyze(t *testing.T) {
 	tests := []struct {
-		name      string
-		inputFile string
-		want      *analyzer.AnalysisResult
-		wantErr   string
+		name            string
+		firstInputFile  string
```

Comment:
make it a collection. (to handle a case when third file is required)

Reply:
Done